app/screens/conversion-flow/conversion-details-screen.tsx

```diff
@@ -349,16 +369,34 @@ export const ConversionDetailsScreen = ({
                 </>
               )}
             {fromWalletCurrency === WalletCurrency.Usd && (
-              <FakeCurrencyInput
-                value={moneyAmountToMajorUnitOrSats(usdAmount)}
-                onChangeValue={(value) => setAmountsWithUsd(Number(value) * 100)}
-                prefix={fiatSymbol}
-                delimiter=","
-                separator="."
-                precision={2}
-                style={styles.walletBalanceInput}
-                minValue={0}
-              />
+              <>
+                <FakeCurrencyInput
+                  value={moneyAmountToMajorUnitOrSats(usdAmount)}
+                  onChangeValue={(value) => setAmountsWithUsd(Number(value) * 100)}
+                  prefix={"$"}
+                  delimiter=","
+                  separator="."
+                  precision={2}
+                  style={styles.walletBalanceInput}
+                  minValue={0}
+                />
+                <FakeCurrencyInput
+                  value={
+                    convertMoneyAmount
+                      ? moneyAmountToMajorUnitOrSats(
+                          convertMoneyAmount(usdAmount, DisplayCurrency),
+                        )
+                      : NaN
+                  }
+                  prefix={fiatSymbol}
+                  delimiter=","
+                  separator="."
+                  precision={2}
```

Comment:
Precision should come from the `useDisplayCurrency`.

Reply:
yes

Reply:
ok. I'll fix that once it's available from the gql api. because it's in the cache so there is limited information available and don't want to import a hooks here (not sure I even can)

Reply:
done